### ISSUE TITLE
Blurring an application reads the previous anchor

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5292]
**Reported by Bogdana on 2015-08-18 05:14**
Hi guys,

While developing an accessibility compliant site for one of our biggest clients, we faced an issue which I believe is a bug of NVDA. I will give you a short example:

```
<a href="#">Link 1</a>
<div role="application">
<a href="#">Link 2</a>
</div>
<a href="#">Link 3</a>
```

If the focus is on Link 1, then you press tab, NVDA reads “application link 2”, which is fine. The issue is when you press tab again – the focus is being placed on Link 3, but NVDA announces “Link 1 Link3”. Link 1 is being announced despite of the fact that it has not been focused. The event when this occurs is blur – i.e. when you blur outside of an application, NVDA reads the first previous anchor, and I cannot get the logic here.

Any thoughts on the matter will be appreciated.

**Comment by leonardder** [comment]
Confirmed that this issue still occurs with NVDA 2017.4

**Comment by scottaohara** [comment]
Was about to file an issue for this but GitHub's autocomplete results made me aware of this one instead :)

Just for an additional test case: https://codepen.io/scottohara/pen/vPXKMd

**Comment by Adriani90** [comment]
Closing as duplicate of #6687 which is newer and contains a bit more technical info.
